plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -167,6 +189,13 @@ def sweep(self, arguments: List[str]) -> None:
                     num_trials_left = -1
                     break
 
+                if is_search_space_exhausted:
+                    log.info(
+                        "Ax has exhausted the search space and raised "
+                        f"the exception: {search_space_exhausted_exception_msg}"
+                    )
```

Comment:
If this is business as usual, why even tell the user about it?

Reply:
It may not be apparent to the end user that the search terminate early because the search space was exhausted.

Reply:
Ok.
Saying that this is an exception seems misleading still.
I understand that this is the underlying API, but the user message can just say that the search space is exhausted.
if the message from Ax is useful maybe we can use it directly.

Reply:
Hmm good point. I will wait for the follow up to https://github.com/facebookresearch/hydra/pull/1387#issuecomment-778514608 and if Ax will still raise the exception, I will update the message here.